personal_assistant/pa_contacts/views.py

```diff
@@ -84,3 +84,17 @@ def create(request):
 def delete(request, contact_id):
     Contact.objects.get(pk=contact_id).delete()
     return redirect(to='pa_contacts:main')
+
+
+def edit(request, contact_id):
+    contact = get_object_or_404(Contact, id=contact_id)
+    
+    if request.method == 'POST':
+        form = ContactsForm(request.POST, instance=contact)
+        if form.is_valid():
+            form.save()
+            return redirect('pa_contacts:main')
+    else:
+        form = ContactsForm(instance=contact)
+    
+    return render(request, 'pa_contacts/edit.html', {'form': form, 'contact': contact})
```

Comment:
Не знайшов, щоб 2-ге значення з контексту якось використовувалося в шаблоні тож можна його і не передавати:
```suggestion
    return render(request, 'pa_contacts/edit.html', {'form': form})
```